docs/content/preview/yugabyte-platform/back-up-restore-universes/schedule-data-backups.md

```diff
@@ -27,79 +27,6 @@ To schedule backups, backups must be enabled for the universe. On the universe *
 
 Before scheduling a backup of your universe data, create a policy, as follows:
```

Comment:
One comment- This is GA in 2025.1. Not sure if preview (2.25) also should have the change. Otherwise LGTM.